### ISSUE TITLE
Deadly signal does not always terminate the process

### ISSUE THREAD

**Issue by k7sleeper** [mrbar42/node-graceful#10]
Emitting a deadly signal (i.e. `process.emit('SIGTERM')`) does not always terminate the process, only when listening to that deadly signal:
```
Graceful.on('SIGTERM', function(done) {
  console.log('gracefully shutting down');
  done();
});
```
If you just listen to `'exit'`
```
Graceful.on('exit', function(done) {
  console.log('gracefully shutting down');
  done();
});
```
the process does **not** terminate if the event queue is not empty.
I wanted to have my 'exit' listener to be a "catch all listener" but that does not work.

Is this expected behavior?

I'm working under Windows!

**Comment by krazibit** [comment]
This is because `exit` signal is not treated as deadly here https://github.com/mrbar42/node-graceful/blob/master/index.js#L104
@mrbar42 is this on purpose or missed?

**Comment by mattgodbolt** [comment]
Looks like it has been fixed in that line you link, but is not yet released in the npm 0.3.1 version.

**Comment by mrbar42** [comment]
fixed and deployed, thanks guys!
